etc/cp-nifi/values.yaml

```diff
@@ -0,0 +1,39 @@
+properties:
+  isNode: true
+
+ingress:
+  enabled: true
+  path: /nifi
+  className: nginx
+  annotations:
+    nginx.ingress.kubernetes.io/backend-protocol: HTTPS
```

Comment:
Please add the cert-manager annotation as exampled in here
https://github.com/RADAR-base/RADAR-Kubernetes/blob/dev/etc/graylog/values.yaml#L29